src/FhirImporter/README.md

```diff
@@ -29,4 +29,40 @@ For a single command line deployment of the container instance:
 
 ```
 az container create --resource-group <RESOURCE GROUP NAME> --image reponame/fhirimporter --name fhirimporter1 --cpu 2 --memory 2 --environment-variables APPINSIGHTS_INSTRUMENTATIONKEY='<KEY>' Audience='<e.g. https://azurehealthcareapis.com>' Authority='<<e.g. https://login.microsoftonline.com/TENANT-ID>' AzureWebJobsDashboard='<STORAGE ACCOUNT CONNECTION STRING>' AzureWebJobsStorage='<STORAGE ACCOUNT CONNECTION STRING>' ClientId='<CLIENT ID>' ClientSecret='<CLIENT SECRET>' FhirServerUrl='<e.g. https://myaccount.azurehealthcareapis.com>' MaxDegreeOfParallelism=16
-```
\ No newline at end of file
+```
+
+There is also a template [azuredeploy-aci-importer.json](../../deploy/templates/azuredeploy-aci-importer.json), which will deploy a storage account, application insights and the function in an Azure Container Instance. Deploy that template with:
+
+```
+ az group deployment create --resource-group <RESOURCE-GROUP-NAME> --template-file .\azuredeploy-aci-importer.json --parameters @aci-template.parameters.json
+```
+
+where `aci-template.parameters.json` is a parameter file with the following contents:
+
+```json
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+       "appName": {
+           "value": "my-aci-importer-app"
+       },
+       "aadAuthority": {
+           "value": "https://login.microsoftonline.com/<TENANT-ID>"
```

Comment:
If sovereign cloud, the URI will be different.

Reply:
Added some more comments.